ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -2358,6 +2358,32 @@ def test_tensorflow_less(
     )
 
 
+# angle
+@handle_frontend_test(
+    fn_tree="tensorflow.math.angle",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
The `tf.math.angle` function should be tested against `complex (or real)` inputs. Can you kindly change this from `"float"` to `"numeric"`?

Reply:
In that case shouldn't I test it with `complex` instead of `numeric`?

Reply:
@AwkNinja `complex` is a subset of `numeric` which also has `float` dtypes which we need to test as well.